### PR TITLE
fix: wrong path to custom.css

### PR DESCRIPTION
**Pull request by Romkabouter**
When trying to use custom.css, that add-on failed to start due to that file missing.
Also, the path was incorrect. 
I think I have updated all necessary files, but please check.

### CODE REVIEW

File: addon/rootfs/etc/cont-init.d/tile-board.sh [hunk 1]
```diff
@@ -13,7 +13,9 @@ do
     FILENAME=$(basename $f);
     if [ $FILENAME = 'custom.css' ]
     then
-        rm /var/tileboard/styles/custom.css
+        if [ -f "/var/www/tileboard/styles/custom.css" ]; then
+            rm /var/www/tileboard/styles/custom.css
+        fi
```
Comment (rchl) [inline addon/rootfs/etc/cont-init.d/tile-board.sh:18]: Can you use `rm -f path` instead? Should not raise the error if the file is missing then.

File: addon/config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "TileBoard",
-  "version": "1.0.6",
+  "version": "1.0.7",
```
Comment (rchl) [inline addon/config.json:3]: You should not modify the version or the changelog. That is done automatically on release.

**Review by rchl — CHANGES_REQUESTED** [review]

File: addon/rootfs/etc/cont-init.d/tile-board.sh (hunk 1 above)
Comment (rchl) [reply addon/rootfs/etc/cont-init.d/tile-board.sh:18]: Well, nothing wrong with the way you did it so I'll resolve it.

File: addon/config.json (hunk 2 above)
Comment (Romkabouter) [reply addon/config.json]: ok, I have reverted this

Comment (rchl) [reply addon/config.json]: There are still changes to package.json and change log

Comment (Romkabouter) [reply addon/config.json]: done

**Review by rchl — APPROVED** [review]

**Pull request merged**